schedule/yam/agama_separate_boot.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: agama_separate_boot
+description: >
+    Prepare auto-installation with home and boot on whole disk.
```

Comment:
don't use auto-installation anymore, but `unattended`. Description should be:
"Installation using directly whole disk for home and boot partition in a different disk"

Reply:
Changed and thanks.